File: ch02/ch02_13.py
with open('./output/col1_col2.txt', 'w') as f:
    with open('col1.txt', 'r') as col1_f:
        with open('col2.txt', 'r') as col2_f:
            for col1, col2 in zip(col1_f, col2_f):
                f.write(col1.strip() + '\t' + col2)

File: ch02/ch02_13.rb
File.open('./output/col1_col2.txt', 'w') do |out_line|
  File.foreach('col1.txt').zip(File.foreach('col2.txt')) do |col1_line, col2_line|
    out_line << [col1_line.strip, col2_line].join('\t')
  end
end
